db/seeds.rb

```diff
@@ -10,32 +10,41 @@
 
 RootNode.create state: 'published'
 
+news = Section.find_or_create_by!(name: "news")
+
 topic = Topic.find_or_create_by!(name: "Business")
 topic.summary = 'The business section covers a range of business-related topics.'
 topic.save
 
+news1 = NewsArticle.find_or_create_by!({
+                                           name: "Business News",
+                                           section: topic,
+                                           parent: news.home_node,
+                                           state: :published,
+                                       })
+news1.revise!(content_body: 'foobar')
```

Comment:
I think seeds.rb needs to be re-evaluated massively. Right now it's mainly full of stuff that should never be run on prod - our rake tasks can tend to that. We might be better served by a prod and dev seeds later on, so that our prod seeds are idempotent and don't add garbage to our db. I'll raise a card for this, but for now I think it's safest if we treat seeds.rb as not-safe-for-prod.